### ISSUE TITLE
Output to schematics/commandblocks to allow for use in vanilla Minecraft

### ISSUE THREAD

**Issue by vopi181** [itsFrank/MinecraftHDL#7]
I mentioned this on Reddit and I know it is a relatively big feet that would require some reorganizing, but I wanted to create an issue so if anyone was interested the could work on it.

**Comment by itsfrank** [comment]
Yeah, definitely not a possibility for me right now, if I were to add to this project I would start with supporting sequential circuits or making it compatible with SCM first. However this is an interesting idea
